packages/ek-components/package.json

```diff
@@ -14,17 +14,22 @@
     "bootstrap": "^4.6.1",
     "bootstrap-vue": "^2.22.0",
     "core-js": "^3.6.5",
+    "kolibri": "^0.16.0-dev.3",
```

Comment:
It’s already technically needed for the babel config.

I have no idea what I’m talking about, but would this be an appropriate use of a [`peerDependency`](https://nodejs.org/en/blog/npm/peer-dependencies)? ek-components could add kolibri/kolibri-tools as a `peerDependency` rather than a `dependency`, then it should be provided by the top-level module.

Reply:
It does sound like a use case for `peerDependency`. I.e., if you were to install the `ek-components` standalone, you'd get `kolibri` with it. However, if (as we are) using `ek-components` as a dependency, it will just ensure that the parent has an appropriate version. That seems reasonable to me.

Reply:
Hmm, actually I see that this would ripple into `template-ui`. Currently `template-ui` doesn't depend on kolibri presumably so that it doesn't have to be included in the channel zips and then it just resolves the needed APIs from `window.kolibri` at runtime. Is that correct? I suppose that's what #784 vs #790 is about.

Reply:
Yes, the per-channel custom presentations built from the template are supposed to interact with the plugin through the API provided as `window.kolibri`. We even upstreamed that at one point https://github.com/learningequality/kolibri/blob/d40c0a37cddc3b323c782d93fe91b221e2d0a5fb/packages/hashi/README.md?plain=1#L26-L35. The idea was to ship each presentation as a ZIP file inside the channel data and upload it to Kolibri Studio. But since we can't manage channels like TED-Ed, the ZIPs are still bundled within Android/Windows/(flatpak?)

Reply:
I think this can be dropped with your latest commits to use `window.kolibri.createTranslator`, right?